packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -264,10 +264,10 @@ export async function grantRoles(
 export async function retrieveCloudSQLConnectionInfo(celoEnv: string, instanceName: string) {
   await validateExistingCloudSQLInstance(instanceName)
   const [blockscoutDBUsername] = await execCmdWithExitOnFailure(
-    `kubectl get secret ${celoEnv}-blockscout --export -o jsonpath='{.data.DB_USERNAME}' -n ${celoEnv} | base64 --decode`
+    `kubectl get secret ${celoEnv}-blockscout --export -o jsonpath='{.data.DATABASE_USERNAME}' -n ${celoEnv} | base64 --decode`
```

Comment:
This should be `DATABASE_USER`, ran into this issue when redeploying using this branch

Reply:
👍 Already fixed by Nam

Reply:
I changed it to DATABASE_USERNAME which was wrong as well, should be `DATABASE_USER` as @tkporter suggested

Reply:
Right. Updated.